src/osd/scrubber/pg_scrubber.cc

```diff
@@ -538,6 +538,7 @@ void PgScrubber::update_scrub_job(const requested_scrub_t& request_flags)
       m_pg->info,
       m_pg->get_pgpool().info.opts);
     m_osds->get_scrub_services().update_job(m_scrub_job, suggested);
+    m_pg->publish_stats_to_osd();
```

Comment:
Is moving `publish_stats_to_osd()` related to this change?

Reply:
Yes - as part of the improvement here was including the "duration of being blocked" in the
PG dump output. Now - there is a 2'nd PR that makes those updates less sporadic. But the update here is a must for the first update